### ISSUE TITLE
Build getting failed after calling "swipeViewWithAccessibilityLabel" ?

### ISSUE THREAD

**Issue by faisalazeez** [kif-framework/KIF#1035]
I have a view, where I have added a gesture for the swipe. I did set accessibilityLabel for the view. but however, my test fails whenever I write "swipeViewWithAccessibilityLabel". Here's my code,

    [self fillIn:@"SignUp_FirstName" text:@"firstname"];
    [self fillIn:@"SignUp_LastName" text:@"lastname"];
    [self fillIn:@"SignUp_Email" text:@"abc@gmail.com"];
    [self fillIn:@"SignUp_Password" text:@"Abc123!"];
    [tester tapViewWithAccessibilityLabel:@"SignUp_CheckButton"];
    [tester tapViewWithAccessibilityLabel:@"SignUp_Join"];
    [tester waitForViewWithAccessibilityLabel:@"Signing Up"];
    [tester waitForViewWithAccessibilityLabel:@"Log In"];
    [tester swipeViewWithAccessibilityLabel:@"swipe" inDirection:KIFSwipeDirectionRight]
